libsolidity/interface/CompilerStack.cpp

```diff
@@ -695,6 +695,31 @@ bool onlySafeExperimentalFeaturesActivated(set<ExperimentalFeature> const& featu
 			return false;
 	return true;
 }
+
+bytes createCBORMetadata(string _metadata, bool _experimentalMode)
```

Comment:
I'm not convinced this should be a private member of the class, but `metadata()` is so we could move this too.

Reply:
Do you mean ``CompilerStack::metadata(std::string const& _contractName)``? Because this is public, so I don't quite understand your comment ;) Also, could we move the anonymous namespace (including this function) to the top of the file? In general I could see this also been implemented in a separate utility namespace.

Reply:
Yes, it should probably be a private member.

Reply:
@bit-shift I meant `createMetadata`.